frontend/app/src/components/home/ChatMessage.svelte

```diff
@@ -891,11 +892,6 @@
         }
     }
 
-    .username {
```

Comment:
Is this intentional - seems like the element with this class is still there

Reply:
I don't think this will take effect right? There is already css for .username scoped to .message-bubble.